### ISSUE TITLE
Update Project Profile: Home Unite Us Remove Farah Khan

### ISSUE THREAD

**Issue by essencegoff** [hackforla/website#7477]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/home-unite-us.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x]  Find the `leadership` variable and remove the following:
```
  - name: Farah Khan
    github-handle:
    role: UX/UI Designer
    links:
      slack: "https://hackforla.slack.com/team/U01JC8VEKMH"
      github: "https://github.com/fourmatte"
    picture: https://avatars.githubusercontent.com/fourmatte
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Home Unite Us page [^2]
- [x] Once your pull request is merged, go to #7362  and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #7362  to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #7362 
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/home-unite-us

**Comment by HackforLABot** [comment]
Hi @siyunfeng, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by siyunfeng** [comment]
- Availability: Weekdays (6:00 pm - 10:00 pm, EDT)
- ETA: Oct 18 (once PR is merged)
